docs/reporting/automating-report-generation.asciidoc

```diff
@@ -2,7 +2,7 @@
 [[automating-report-generation]]
 == Automating Report Generation
 You can automatically generate reports with {watcher}, or by submitting
-HTTP POST requests from a script.
+HTTP `POST` requests from a script.
```

Comment:
I felt this page needs consistency on how the word "POST" is formatted. I decided a good way to go would be to use proper-noun casing for "POST URL" and use code formatting for when `POST` refers to the HTTP method.

Reply:
@tsullivan I agree with POST URL, but it looks like in other areas of the Kibana docs, we use 'POST' HTTP. For example, https://www.elastic.co/guide/en/kibana/current/kibana-authentication.html#saml. For the sake of consistency, could we use the same format? Or if this is wrong, we should do a pass through the docs and correct these instances.

Reply:
Thanks! If I understand correctly, I think as you've suggested is the way I have it. 

"POST URL" isn't back-ticked because it's the name of the string that's how we refer to it in the UI:
![image](https://user-images.githubusercontent.com/908371/61319319-d7a2bb80-a7bb-11e9-9f9a-3fe89dd696ae.png)

"'POST' HTTP" (with single quotes just around POST) is not how I see it in the link you gave. Those instances of the word are backticked - not singlequoted - giving them the code-format look:
![image](https://user-images.githubusercontent.com/908371/61319490-2e0ffa00-a7bc-11e9-92e0-a73ebb985e8d.png)

Let me know if I have it OK :D

Reply:
My apologies for the confusion, but I agree that back ticks are necessary here, not single quotes. Thank you for pushing these changes!